conf/nginx.conf

```diff
@@ -12,12 +12,15 @@ server {
 
     # When requesting static paths with an extension, try them, and return a 404 if not found
     location ~* .+\..+ {
+        add_header Cache-Control "public, s-maxage=300, max-age=60, must-revalidate";
         try_files $uri $uri/ =404;
     }
 
     # When requesting a path without extension, try it, and return the index if not found
     # This allows for client-side route handling
     location / {
-        try_files $uri $uri/ /index.html$is_args$args;
+        # Ensure Cloudflare caches assets for 5 minutes
+        add_header Cache-Control "public, s-maxage=300, max-age=60, must-revalidate";
+        try_files $uri $uri/ /index.html;
```

Comment:
I have removed `$is_args$args`, because those actually make no sense when loading a static file. They are meant to be used in fast-cgi contexts, like with PHP and such, where there might be some server-side logic run before nginx returns the result.